pkgs/tools/system/kanata/default.nix

```diff
@@ -28,7 +37,7 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/jtroo/kanata";
     license = licenses.lgpl3Only;
     maintainers = with maintainers; [ linj ];
-    platforms = platforms.linux;
+    platforms = platforms.all;
```

Comment:
if you want linux+darwin, that's `platforms.unix`.

Reply:
Upstream supports windows as well.  However, I am not sure how to deal with windows in Nixpkgs.

On `unix` vs `all`, I find that the manual [says][1]:

> In general it is preferable to set `meta.platforms = lib.platforms.all` and then exclude (using `badPlatforms`) any platforms on which the package is known not to build

[1]: https://nixos.org/manual/nixpkgs/unstable/#var-meta-badPlatforms

Reply:
Ah, from the PR description I got just "added darwin". If others are supported, then all good 👍

Reply:
I intended to only add darwin support since I don't have any way to test that windows can actually build the package. I am happy to leave `platforms.all`, but I don't want to increase the support burden if it will be completely untested.